crates/graphql/src/schema/query_root.rs

```diff
@@ -57,6 +63,9 @@ impl QueryRoot {
     ) -> FieldResult<Vec<Wallet>> {
         let conn = ctx.shared.db.get().context("failed to connect to db")?;
 
+        // TODO temporarily force specific wallets
```

Comment:
Note this needs to be updated to include only wallets we want to surface _before_ merging. It will affect existing queries from the frontend!

@kespinola @kristianeboe

Reply:
nm, I was able to copy it over from the frontend code